### ISSUE TITLE
No need to stringify POST data

### ISSUE THREAD

**Issue by HTEredrock** [inorganik/digest-auth-request#31]
This is more of a FYI than an issue, but thought I'd let you know.

In your readme, it states that for post data you create an object variable and convert it to a string.

```
var postData = {
   address: '123 main st.',
   city: 'Denver',
   state: 'Colorado'
}
postData = JSON.stringify(postData);
```

This however is unnecessary since I see your code already does this to the passed in data variable.

```
        // start here
        // successFn - will be passed JSON data
        // errorFn - will be passed error status code
        // data - optional, for POSTS
        this.request = function(successFn, errorFn, data) {
            // posts data as JSON if there is any
            if (data) {
                self.data = JSON.stringify(data);
            }
            self.successFn = successFn;
            self.errorFn = errorFn;

            if (!self.nonce) {
                self.makeUnauthenticatedRequest(self.data);
            } else {
                self.makeAuthenticatedRequest();
            }
        }
```

Anyways, thanks for the awesome piece of code!

**Comment by inorganik** [comment]
Good call, thanks!

**Comment by inorganik** [comment]
Fixed in 0.8.0
